content/blog/2024/11-01_lapply_files/index.qmd

```diff
@@ -153,6 +186,8 @@ Now, I really struggle to explain `do.call` because it can do several things dep
 In all honestly, the only scenario I can use the function is this exact one.
 How it works in my head is that it takes all the elements in the list you provide and applies the function you provide it to them all.
 
+Out of curiousity, my friend [Maëlle](https://masalmon.eu/) had a [search on GitHub](https://github.com/search?q=+language%3AR+do.call%28&type=code&ref=advsearch) and found that at least on public repos on GitHub, my usecase is quite common.
```

Comment:
hehe GitHub search for the win. Might be worth linking to Jenny's stuff on GitHub searches as a footnote?https://github.com/posit-conf-2024/pkg-dev/blob/main/materials/pkg-dev-posit-conf-2024-introduction.pdf